browser/brave_wallet/solana_provider_renderer_browsertest.cc

```diff
@@ -128,11 +130,11 @@ constexpr char OnAccountChangedScript[] =
 constexpr char CheckSolanaProviderScript[] = "!!window.braveSolana";
 constexpr char OverwriteScript[] = "window.solana = ['test'];window.solana[0]";
 
-std::string VectorToArrayString(const std::vector<uint8_t>& vec) {
+std::string VectorToArrayString(base::span<const uint8_t> data) {
   std::string result;
-  for (size_t i = 0; i < vec.size(); ++i) {
-    base::StrAppend(&result, {base::NumberToString(vec[i])});
-    if (i != vec.size() - 1) {
+  for (size_t i = 0; i < data.size(); ++i) {
+    base::StrAppend(&result, {base::NumberToString(data[i])});
+    if (i != data.size() - 1) {
       base::StrAppend(&result, {", "});
     }
```

Comment:
nit: this can be rewritten with a bool that prepends `", "` if it's not the first item. This way you can replace for-loop with a simplified ranged version.

or maybe just this:
```cpp
return base::JoinString(
    base::ToVector(data, [](int value) { return base::NumberToString(value); }), 
    ", ");
```

Reply:
Fixed. I'm going with the `JoinString` version because it is easier to the reader.